testcases/open_posix_testsuite/conformance/interfaces/free/1-1.c

```diff
@@ -0,0 +1,78 @@
+/*
+ * This file is licensed under the GPL license. For the full content
+ * of this license, see the COPYING file at the top level of this
+ * source tree.
+ */
+
+/*
+ * assertion:
+ *
+ *	The function will cause the space pointed to by ptr to be deallocated.
+ *
+ * method:
+ *	-Set rlimit equals to 1024 * 1024 * 2.
+ *	-Allocate memory to pointer_1 of size 1024 * 1024 *1.
+ *	-Alloacte memory for pointer_2 of size 1024 *1024 *1.
+ *	But it will fail due to insufficient memory.
+ *	-Try allocating memory to pointer_2 after freeing memory
+ *	allocated to pointer_1.
+ *	-Check whether memory is allocated by malloc().
+ *
+*/
+
+#define _XOPEN_SOURCE 700
```

Comment:
The explicit definitions of XOPEN_SOURCE were removed in 310c9eb6e6af321c7515b1fc59df73c2a5b7c084 new tests must not include these.